bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/aemform/AEMForm.java

```diff
@@ -144,6 +151,15 @@ default FormType getFormType() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns the theme path
+     *
+     * @return the role attribute, if one was set, or {@code null}
+     * @since com.adobe.cq.forms.core.components.models.aemform 1.5.0;
+     */
+    @JsonIgnore
+    String getRoleAttribute();
```

Comment:
Please add default implementation for this

Reply:
Earlier I added the default implementation but coverage is dropping because of which changed into this. Test case is not able to cover the default implementation tried that also. Other default implementation in this are also not covered.
Updated the java doc.